### PR TITLE
Update reader-pcsc.c

### PR DESCRIPTION
**Pull request by hajikhorasani**
Error handling extended in case of changing the card reader

##### Checklist
- [x] PKCS#11 module is tested
- [x] Windows minidriver is tested

### CODE REVIEW

**Comment by hajikhorasani** [comment]
Now, it's possible to detach and attach the card readers without unload and load the library.

File: src/libopensc/reader-pcsc.c [hunk 1]
```diff
@@ -339,6 +339,16 @@ static int refresh_attributes(sc_reader_t *reader)
 			reader->flags &= ~SC_READER_CARD_CHANGED;
 			LOG_FUNC_RETURN(reader->ctx, SC_SUCCESS);
 		}
+		
+		/* the system could not dectect any reader. It means, the prevoiusly attached reader is disconnected. */		
+		if (rv == SCARD_E_NO_READERS_AVAILABLE || rv == SCARD_E_SERVICE_STOPPED) {
```
Comment (frankmorgner) [inline src/libopensc/reader-pcsc.c]: cast to (LONG)

**Comment by hajikhorasani** [comment]
Yes, it’s reproducible. I see the problem usually with Gemalto card readers with embedded SIM-sized smart cards.

**Comment by frankmorgner** [comment]
how can it be reproduced?

though you've *resolved* the conversation above, the build problem persists...

**Comment by hajikhorasani** [comment]
I’ll resolve the build problem ASAP.

**Comment by hajikhorasani** [comment]
@frankmorgner I've resolved the build problem, but there is an issue in "Inno Setup" with HOST=mingw. How could I fix it ? Is it related to source code that I changed ?
Error:
wine: cannot find 'C:/Program Files (x86)/Inno Setup 5/ISCC.exe'
The command "if [ ! -z "$HOST" -a "${DO_COVERITY_SCAN}" != "yes" ]; then make install; wine "C:/Program Files (x86)/Inno Setup 5/ISCC.exe" win32/OpenSC.iss; fi" exited with 2.

**Pull request merged**